### ISSUE TITLE
Next release planned date? 

### ISSUE THREAD

**Issue by MattArnold93** [Wikiki/bulma-calendar#280]
Hi there,

I am wondering when the next planned release for the project is? The calendar component currently breaks if the min or max date variables are dynamic, and the fix(es) included in what would be the next release resolve this issue. This bug is of critical importance to me and my team. 

Just looking to understand the planning for this so we can manage expectations for clients. Thanks.

**Comment by michael-hack** [comment]
Hi, I don't currently have a fixed date planned for a next release. I only contribute to this project from time to time as my own time allows.

Currently I'm busy with a customer project and that's why I haven't done much work here in the last weeks. After this project I have another project coming up, where I will need the calendar again and I think that I will be able to bring some new features and fixes.

But that should not stop you and your team, you are welcome to make changes and make a pull request. Or at least look for the cause of the problems and provide a fix in the issues, which I can then add when the time comes.

**Comment by MattArnold93** [comment]
@michael-hack thanks for the response. I don't think I made it clear in my initial message, but the bug fixes I mention above were already merged into master about 3-4 weeks ago (when I made my pull request, I think you merged it actually). All we are waiting on at this point is the official release so we can update the module in our project.

**Comment by michael-hack** [comment]
Oh sorry, I have completely missed this. I will make a new release later....
